src/main/java/hudson/plugins/sonar/SonarInstallation.java

```diff
@@ -138,6 +138,9 @@ public String getServerUrl() {
    * @since 2.4
    */
   public String getServerAuthenticationToken() {
+    if (serverAuthenticationToken != null) {
+      return serverAuthenticationToken.trim();
```

Comment:
Please simplify this method using `return StringUtils.trimToNull(serverAuthenticationToken);`